modules/mobile-agents/android/client/src/org/wso2/mdm/agent/services/AgentDeviceAdminReceiver.java

```diff
@@ -88,26 +88,26 @@ public void onDisabled(Context context, Intent intent) {
 	public void startUnRegistration(Context context) {
 		String regId = Preference.getString(context, context
 				.getResources().getString(R.string.shared_pref_regId));
-
+        	LocalNotification.stopPolling(context);
```

Comment:
This is a service class that is used to start/stop device alarm manager service.